include/capstone/avr.h

```diff
@@ -0,0 +1,231 @@
+    AVR_INS_BRHS,    // Branch if Half Carry Flag Set
+    AVR_INS_BRHC,    // Branch if Half Carry Flag Cleared
+    AVR_INS_BRTS,    // Branch if T Flag Set
+    AVR_INS_BRTC,    // Branch if T Flag Cleared
+    AVR_INS_BRVS,    // Branch if Overflow Flag is Set
+    AVR_INS_BRVC,    // Branch if Overflow Flag is Cleared
+    AVR_INS_BRIE,    // Branch if Interrupt Enabled
+    AVR_INS_BRID,    // Branch if Interrupt Disabled
+    
+    // Data Transfer Instructions
+    AVR_INS_MOV,     // Copy Register
+    AVR_INS_MOVW,    // Copy Register Word
+    AVR_INS_LDI,     // Load Immediate
+    AVR_INS_LD,      // Load Indirect
+    AVR_INS_LDD,     // Load Indirect with Displacement
+    AVR_INS_LDS,     // Load Direct from SRAM
+    AVR_INS_ST,      // Store Indirect
+    AVR_INS_STD,     // Store Indirect with Displacement
+    AVR_INS_STS,     // Store Direct to SRAM
+    AVR_INS_LPM,     // Load Program Memory
+    AVR_INS_ELPM,    // Extended Load Program Memory
+    AVR_INS_SPM,     // Store Program Memory
+    AVR_INS_IN,      // In From I/O Location
+    AVR_INS_OUT,     // Out To I/O Location
+    AVR_INS_PUSH,    // Push Register on Stack
+    AVR_INS_POP,     // Pop Register from Stack
+    
+    // Bit and Bit-test Instructions
+    AVR_INS_SBI,     // Set Bit in I/O Register
+    AVR_INS_CBI,     // Clear Bit in I/O Register
+    AVR_INS_LSL,     // Logical Shift Left
+    AVR_INS_LSR,     // Logical Shift Right
+    AVR_INS_ROL,     // Rotate Left Through Carry
+    AVR_INS_ROR,     // Rotate Right Through Carry
+    AVR_INS_ASR,     // Arithmetic Shift Right
+    AVR_INS_SWAP,    // Swap Nibbles
+    AVR_INS_BSET,    // Flag Set
+    AVR_INS_BCLR,    // Flag Clear
+    AVR_INS_BST,     // Bit Store from Register to T
+    AVR_INS_BLD,     // Bit load from T to Register
+    AVR_INS_SEC,     // Set Carry
+    AVR_INS_CLC,     // Clear Carry
+    AVR_INS_SEN,     // Set Negative Flag
+    AVR_INS_CLN,     // Clear Negative Flag
+    AVR_INS_SEZ,     // Set Zero Flag
+    AVR_INS_CLZ,     // Clear Zero Flag
+    AVR_INS_SEI,     // Global Interrupt Enable
+    AVR_INS_CLI,     // Global Interrupt Disable
+    AVR_INS_SES,     // Set Signed Test Flag
+    AVR_INS_CLS,     // Clear Signed Test Flag
+    AVR_INS_SEV,     // Set Overflow Flag
+    AVR_INS_CLV,     // Clear Overflow Flag
+    AVR_INS_SET,     // Set T in SREG
+    AVR_INS_CLT,     // Clear T in SREG
+    AVR_INS_SEH,     // Set Half Carry Flag in SREG
+    AVR_INS_CLH,     // Clear Half Carry Flag in SREG
+    
+    // MCU Control Instructions
+    AVR_INS_NOP,     // No Operation
+    AVR_INS_SLEEP,   // Sleep
+    AVR_INS_WDR,     // Watchdog Reset
+    AVR_INS_BREAK,   // Break
+    
+    AVR_INS_ENDING
+} avr_insn;
+
+// AVR operand types
+typedef enum avr_op_type {
+    AVR_OP_INVALID = 0, // Uninitialized/invalid operand
+    AVR_OP_REG,         // Register operand
+    AVR_OP_IMM,         // Immediate operand
+    AVR_OP_MEM,         // Memory operand
```

Comment:
```suggestion
    AVR_OP_INVALID = CS_OP_INVALID, // Uninitialized/invalid operand
    AVR_OP_REG = CS_OP_REG,         // Register operand
    AVR_OP_IMM = CS_OP_IMM,         // Immediate operand
    AVR_OP_MEM = CS_OP_MEM,         // Memory operand
```

You need to include `cs_operand.h`